main.py

```diff
@@ -102,7 +104,49 @@ def main():
     stats["execution_time"] = f"{hours_str}, {minutes_str}, {seconds_str}"
 
     print("===>Process completed")
+
```

Comment:
Move this into a separate script, and use argparse to add a command-line parameter which is the number of days to process. Default that parameter to 7.